README.md

```diff
@@ -254,4 +254,108 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
 
     // <SNIP>
 }
-```
\ No newline at end of file
+```
+## How to Use (Xamarin Studio)##
+
+If you're using Xamarin Studio the templates above won't work. This guide assumes that you have a solution with either a PCL or a Shared project and a number of platform specific projects. It doesn't matter if you're using Xamarin.Forms or Xamarin.iOS/Xamarin.Android directly. Your solution might look something like this:
+
+```C#
```

Comment:
Could you remove the C# language specification from all these code blocks? This isn't C#, so should just be displayed as a plain text <pre>.